src/main/java/thePackmaster/patches/SingleCardViewBorderTextPatch.java

```diff
@@ -39,7 +39,7 @@ public static void Postfix(SingleCardViewPopup __instance, SpriteBatch sb) {
             AbstractCard card = (AbstractCard)cardField.get(__instance);
             if (card instanceof AbstractPackmasterCard) {
                 ((AbstractPackmasterCard)card).renderBorderText(sb, Settings.WIDTH / 2.0F, (float)yField.get(__instance), yOffsetBase, drawScale);
-            } else if (AbstractDungeon.player != null && AbstractDungeon.player.chosenClass == ThePackmaster.Enums.THE_PACKMASTER && card.getClass().getSuperclass().equals(AbstractCard.class)) {
+            } else if (RenderBaseGameCardPackTopTextPatches.shouldShowPackName(card)) {
```

Comment:
Previously this had the same condition check as the non-SCV version. That was fine when it wasn't too complex, but with the additional complexity that I introduced, I centralized the logic.